### PR TITLE
support for comma-separated chat channels

### PR DESCRIPTION
**Pull request by npinsker**
Listen in on multiple chat channels at once by separating them with a comma

### CODE REVIEW

**Comment by dashed** [comment]
:+1: Nice...

**Comment by 5a1t** [comment]
I have a separate fork of this working as well.

File: robin.user.js [hunk 1]
```diff
@@ -16,9 +16,14 @@
     GM_addStyle('.robin--username {cursor: pointer}');
 
     // Utils
-    function hasChannel(source, channel) {
-        channel = String(channel).toLowerCase();
-        return String(source).toLowerCase().startsWith(channel);
+    function hasChannel(source, channels) {
+	    for (var i in channels) {
+	        var channel = String(channels[i]).toLowerCase();
```
Comment (dashed) [inline robin.user.js:21]: ~~Move this outside of the loop.~~

EDIT: idk how to read code

**Comment by dashed** [comment]
Add bool option to enable/disable multiple channels in cases when actual channel name needs to contain a comma.

**Comment by devoidfury** [comment]
How does this interact with the auto-prefixer?

**Comment by dashed** [comment]
Good point. I think this PR was created prior to the channel auto-prefixer.

**Comment by vartan** [comment]
It'd be nice to have a select input field which chooses which prefix you're using.

**Comment by dashed** [comment]
Just do a dropdown and put it beside the input textbox.

**Comment by dashed** [comment]
Also, this needs good visual cues to identify which filter an output is coming from.

**Comment by vartan** [comment]
Implemented myself, sorry about that

**Pull request closed without merging**